Sources/Widgets/Representations/ConeHandleRepresentation/index.js

```diff
@@ -6,11 +6,14 @@ import vtkPixelSpaceCallbackMapper from 'vtk.js/Sources/Rendering/Core/PixelSpac
 import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
```

Comment:
is ConeHandleRepresentation really needed ? Can't it be merged into ArrowHandleRepresentation ?

Reply:
The coneHandle is a 3d Glyph. It can be merged with arrowHandle but would break the 3d/2d distinction in arrowHandle vs other glyph/handle representations

Reply:
I feel that a ConeHandleRepresentation will not be used by any other widget. But only as an "arrow" handle. 
Eventually we could create a "3D arrow handle representation" but is there a need ? Can't ArrowHandleRepresentation support 3D shapes ?